ospd/misc.py

```diff
@@ -868,6 +878,9 @@ def filename(string):
                         help='Path to the logging file.')
     parser.add_argument('--version', action='store_true',
                         help='Print version then exit.')
+    parser.add_argument('--niceness', default=NICENESS, type=niceness,
```

Comment:
Is there any reason for defining an own type instead using the default int type https://docs.python.org/3/library/argparse.html#type ?